components/trades/NewTrade.tsx

```diff
@@ -139,6 +139,7 @@ export default function NewTrade({
       quickness: 1,
       control: 1,
       conditioning: 1,
+      playerID:0,
```

Comment:
Do you not have Format on Save enabled? Not that I care about spacing here, it doesn't matter

Reply:
i do not, ill look into it